lib/suma/admin_api/common_endpoints.rb

```diff
@@ -30,6 +53,13 @@ def update_model(m, orig_params, process_params: nil, save: true)
             fk_attrs[assoc[:name]] = Suma::TranslatedText.find_or_create(**v)
           elsif association_class?(assoc, Suma::Address)
             fk_attrs[assoc[:name]] = Suma::Address.lookup(v)
+          elsif association_class?(assoc, Suma::LegalEntity)
```

Comment:
This isn't right either. Don't add code to a general helper, that only makes sense given some specific context you can't infer from the use case. Solve it some other way.